### ISSUE TITLE
Investigate possibility to be dependent on external packages

### ISSUE THREAD

**Issue by nvborisenko** [reportportal/commons-net#40]
Now our direction is to be dependent on external packages less as possible. The goal of this is minimize conflicting of user's dependencies.

From other point we want to use dependencies to speed up development or package performance. The problem is all dependent assemblies should be shipped into a folder where agent is located to be properly loaded by .Net Runtime.

We need investigate whether runners properly load reporters with dependencies.

**Comment by nvborisenko** [comment]
Some time ago I decided to exclude 3rd party dependencies because of there were problems to keep dependencies near with main assembly. It was good decision at that time.

Now our focus is shipping agents via nuget packages, and it is promising to get an ability to use external nuget packages. The issue is we need understand whether msbuild tooling really copies dependencies to the output directory. I saw that some version of msbuild doesn't do it, or it's dependent on tests project? not clear, .netcore2.0 does like that?

Here we need understand the audience what don't copy dlls to output folder by default. And remove msbuild tasks which copies our dependencies to output folder.
